yaqc_cmds/somatic/acquisition.py

```diff
@@ -50,33 +50,26 @@
 
 
 class Axis:
-    def __init__(self, points, units, name, identity, hardware_dict={}, **kwargs):
+    def __init__(self, points, units, name, hardware_dict={}, **kwargs):
         self.points = points
         self.units = units
         self.name = name
-        self.identity = identity
         self.hardware_dict = hardware_dict.copy()
         self.__dict__.update(kwargs)
         # fill hardware dictionary with defaults
-        names = re.split("[=F]+", self.identity)
-        # KFS 2018-12-07: Is this still used at all? replacing wt2 kit.parse_identity
-        if "F" in self.identity:  # last name should be a 'following' in this case
-            names.pop(-1)
+        # Not sure if this re.split does what we want now, getting rid of identity
+        # KFS 200-10-13
+        names = re.split("[=F]+", self.name)
```

Comment:
Why is this change being made to recover motortune? Isn't this very delicate surgery?

Reply:
It isn't all that delicate, actually... we don't use `identity`, and the only place it had been used really was in motortune, hence why it is related

Reply:
I think this particular line really just needs to be splitting on "=", not even re split... we have not used F for following in a long time, I don't think... and can't really think of why there would be more than one to require the re `+`

That said, this should _work_ (unless something has a name that starts with `F`)

Reply:
I can agree it should work, I just don't have the bandwidth to say in good faith that I agree with the change.